sphinx/tutorial/rose/furthertopics/rose-stem-tutorial.rst

```diff
@@ -241,24 +241,27 @@ to the prompts)*.
 Running the test suite
 ----------------------
 
-We should now be able to run the test suite. Simply type::
+We should now be able to install the test suite. Simply type::
 
    rose stem --group=command_spaceship
 
 anywhere in your working copy (the ``--source`` argument defaults to ``.``
 so it should automatically pick up your working copy as the source).
 
-.. note::
-
-   If your site uses a Cylc server, and your home directory is not shared
-   with the Cylc server, you will need to add the option::
-
-      --host=localhost
-
 We use ``--group`` in preference to ``--task`` in this suite (both are
 synonymous) as we specify a group of tasks set up in the Jinja2 variable
 ``name_graphs``.
 
+Previously, the ``rose stem`` command ran the suite. At Cylc 8, it simply
```

Comment:
Would the first clause be better in an admonition? 
If you have come fresh to Cylc 8 I'm not sure that this would be helpful...